Classes/CardIOVideoFrame.mm

```diff
@@ -133,8 +133,8 @@ - (void)detectCardInSamplesWithFlip:(BOOL)shouldFlip {
     dmz_transform_card(self.dmz, self.ySample.image, self.corner_points, frameOrientation, false, &foundCardY);
     self.cardY = [CardIOIplImage imageWithIplImage:foundCardY];
```

Comment:
Should this assignment to `self.cardY` also wait until we've confirmed that `foundCardY` is valid?

I.e., if we get as far as the call to `dmz_transform_card()`, but the result is invalid, then act as if `foundCard` were false.